### ISSUE TITLE
Option/change to run all files in a folder

### ISSUE THREAD

**Issue by jakubbrezonak** [jacob135135/Log2SubBand-evaluation#41]
Need to add an option or just change it so that user can select a folder and all csv files in that folder will be run. The file with results will be named based on folder name.

I only need to give compression rate of log2 SubBand, Huffman using given codebook and potentially optimal Huffman (if so I need to create a new issue for that).

Some indication of progress would be really good.

**Comment by jakubbrezonak** [comment]
If user selects folder, the effectively #31 is done automatically with help of #54

**Comment by jakubbrezonak** [comment]
Not sure how to select folder, however running all files in the same directory with appropriate extension if appropriate checkbox is set should do the trick.

**Comment by jakubbrezonak** [comment]
Once all appropriate files are detected, excluding special (e.g. codebook) files, they need to run sequentially, recording their stats and outputting corresponding files. 
Once that all is done, it would be good to give overall results file (assuming at least 2 csv input files were used)

**Comment by jakubbrezonak** [comment]
0eb1d086bb307c86c692022827765abcdf38245f
